sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -1353,7 +1353,7 @@ protected sealed override BlobBaseClient WithCustomerProvidedKeyCore(CustomerPro
         /// been committed.  These blocks are stored in Azure in association
         /// with a blob, but do not yet form part of the blob.
         /// </summary>
-        /// <param name="listType">
+        /// <param name="listTypes">
```

Comment:
Why not `blockListTypes`?

Reply:
I think that is fine.. Was pretty much just going with what was already there.